### PR TITLE
ENH: Add crosshair toggling to OrthoSlicer3D

### LINKED ISSUE #689: Disable crosshairs OrthoSlicer3D
Hello, how I can disable the crosshairs of Orthoslicer3D?

thank you.

### PR DESCRIPTION
**Pull request by mko010**
Add a toggle for crosshairs with ctrl+x

Closes #689.

### CODE REVIEW

File: nibabel/viewers.py [hunk 1]
```diff
@@ -498,8 +502,11 @@ def _draw(self):
         for ii in range(3):
             ax = self._axes[ii]
             ax.draw_artist(self._ims[ii])
-            for line in self._crosshairs[ii].values():
-                ax.draw_artist(line)
+            if self._cross:
+                for line in self._crosshairs[ii].values():
+                    ax.draw_artist(line)
+            else:
+                pass
```
Comment (effigies) [inline nibabel/viewers.py]: No need for `else: pass`.

**Comment by mko010** [comment]
Thanks for the review, I will review it tomorrow, or when I have time to do it.

Have a good day.

El vie., 21 dic. 2018 23:36, Chris Markiewicz <notifications@github.com<mailto:notifications@github.com>> escribió:

@effigies commented on this pull request.

Hi, sorry about the slow review. I think this is almost good to go, but on another look through, you can drop a branch.

________________________________

In nibabel/viewers.py<https://github.com/nipy/nibabel/pull/701#discussion_r243706510>:

> @@ -498,8 +502,11 @@ def _draw(self):
         for ii in range(3):
             ax = self._axes[ii]
             ax.draw_artist(self._ims[ii])
-            for line in self._crosshairs[ii].values():
-                ax.draw_artist(line)
+            if self._cross:
+                for line in self._crosshairs[ii].values():
+                    ax.draw_artist(line)
+            else:
+                pass


No need for else: pass.

**Comment by effigies** [comment]
Looks good. Thanks.

**Comment by mko010** [comment]
You are welcome!

El mar., 25 dic. 2018 5:45, Chris Markiewicz <notifications@github.com<mailto:notifications@github.com>> escribió:

Looks good. Thanks.

**Pull request merged**